src/codeu/chat/server/bots/HangmanBot.java

```diff
@@ -0,0 +1,68 @@
+package codeu.chat.server.bots;
+
+public class HangmanBot extends Bot {
```

Comment:
It would be nice to see some tests for this class, especially since it's pretty standalone.